AlfrescoApp/Model/Managers/RealmSyncHelper.m

```diff
@@ -120,6 +120,26 @@ - (NSString *)syncIdentifierForNode:(AlfrescoNode *)node
     return syncIdentifier;
 }
 
+- (NSMutableArray *)syncIdentifiersForNodes:(NSArray *)nodes
+{
+    NSMutableArray *syncIdentifiers = [NSMutableArray array];
+    
+    for (AlfrescoNode *node in nodes)
+    {
+        [syncIdentifiers addObject:[self syncIdentifierForNode:node]];
```

Comment:
Is it worth performing a nil check before adding to the array? In theory `nodeRefWithoutVersionID` could return nil - unlikely, but still possible.